healthcheck/port.conf

```diff
@@ -0,0 +1,5 @@
+server {
+	listen       8080;
+	listen 80;
```

Comment:
Why is it listening on both 80 and 8080 ?

Reply:
only 8080 is useful, I will remove 80